docker/version.txt

```diff
@@ -1 +1 @@
-0.1
+prod
```

Comment:
Before merging, all of the docker images would need new tags. Doing this allows us to define which images are being used for the build without locking in on a particular version number.